### PR TITLE
Add Minecraft Central presence.

### PR DESCRIPTION
**Pull request by Feeeh**
Examples:
image
image
image
image
and much more.

As I said before, I'm pretty new with js/ts and GitHub, so pls don't hate. lul

I think I fixed the tab width. 4 for js/ts and 2 for json files. Any problems, please tell me. :)

### CODE REVIEW

**Comment by doomlerd** [comment]
Recompile the presence with : 
`tsc --noFallthroughCasesInSwitch --noUnusedLocals --noUnusedParameters -t "ES2020"`

**Comment by Feeeh** [comment]
> Recompile the presence with :
> `tsc --noFallthroughCasesInSwitch --noUnusedLocals --noUnusedParameters -t "ES2020"`

Done, I'm sorry. New with all js/ts. ;p
Thanks for the help.

File: Minecraft Central/presence.ts [hunk 1]
```diff
@@ -0,0 +1,447 @@
+var presence = new Presence({
+    clientId: "693097839424831489",
+    mediaKeys: false
+});
```
Comment (Bas950) [inline Minecraft Central/presence.ts]: mediakeys has been fully removed, so please remove it here aswell :)

Comment (Feeeh) [reply Minecraft Central/presence.ts]: Done.
Again, thanks for the help. :)

**Pull request merged**